spec/update/gems/post_install_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+describe "bundle update" do
+  let(:config) {}
+
+  before do
+    gemfile <<-G
+      source "file://#{gem_repo1}"
+      gem 'rack', "< 1.0"
+      gem 'thin'
+    G
+
+    bundle "config #{config}" if config
+
+    bundle :install
```

Comment:
Oh thanks, I hadn't noticed that method.  I put it as a separate commit, not sure what the preference is here :)